README.md

```diff
@@ -238,7 +238,7 @@ Assuming plugin installation has been done already.
      
 6. Adding a Personal Access Token Credentials:
 
-   This is a manual setup. To automatically generate Personal Accees Token see next section
+   This is a manual setup. To automatically generate Personal Accees Token see next [section](https://github.com/baymac/gitlab-branch-source-plugin#creating-personal-access-token-within-jenkins).
```

Comment:
You can do relative links :wink: 
```suggestion
   This is a manual setup. To automatically generate Personal Accees Token see next [section](#creating-personal-access-token-within-jenkins).
```

Should do the trick

Reply:
Cool.

Reply:
https://github.com/baymac/gitlab-branch-source-plugin/blob/8fe081304dec8afbb44842a70671720ddd36046b/README.md yup works :sweat_smile: